README.md

```diff
@@ -39,192 +29,146 @@ After you're finished please remove all the comments and instructions!
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 Catalog of my things <a name="about-project"></a>
 
-**[your_project__name]** is a...
+This project is console app built using ruby, the app will help you to keep a record of different types of things you own: books, music albums, movies, and games, the data stored in json file, and it will be always useable whenever you start the app.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
+- The app built using [Ruby](https://www.ruby-lang.org/en/).
+- The **schema** file built using **SQL**.
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+The app help you to keep recods about any items you own, thing like
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Store and Display Movies**
+- **Store and Display Games**
+- **Store and Display Music Albums**
+- **Store and Display Books**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
+## Video Description <a name="vd-disc"></a>
 
-> Add a link to your deployed project.
+In this recorder video we are describing how we built this amazing project as a team.
 
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+[Link to watch the video](https://drive.google.com/file/d/1p1Hps1mL0TJVm6GKwXbOT19G3V_3lT4s/view?usp=sharing)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
+To get a local copy up and running, follow these steps...
 
 ### Prerequisites
 
 In order to run this project you need:
 
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+- Ruby should be installed in your local machine
+- You need a Terminal.
+- You need a code editor eg: `VsCode`
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
 Example commands:
 
 ```sh
   cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone https://github.com/adel-gu/Catalog-of-my-things.git
 ```
---->
 
 ### Install
 
 Install this project with:
 
-<!--
 Example command:
 
 ```sh
   cd my-project
-  gem install
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you update this `cd my-project` to the correct name of your repository once some has cloned it. `cd Catalog-of-my-things` instead of `cd my-project`

Reply:
Sure I'll do so, thank you so much @tresorsawasawa 👏